### PR TITLE
kpb: lnl: update uuid in rimage for LNL

### PR DESCRIPTION
**Pull request by jxstelter**
Change of uuid in LNL config
regarding Windows compatibility

Signed-off-by: Jaroslaw Stelter <Jaroslaw.Stelter@intel.com>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ manifest:
     - name: rimage
       repo-path: rimage
       path: sof/rimage
-      revision: ab0429fdbe563ef6abe499c69b2483e96c4762d0
+      revision: 4ce79b152e23c7b3937a09ef57cdd69d125a9214
```
Comment (plbossart) [inline west.yml:37]: the GUID was already changed by this https://github.com/thesofproject/sof/pull/7343/commits/b37d466a69a8b8dfb0b1ef42ff164f87b005a4b8

???

Comment (kv2019i) [reply west.yml:37]: @plbossart  That was for MTL, the same change is needed in lnl.toml.

**Review by kv2019i — CHANGES_REQUESTED** [review]
@jxstelter You need to update rimage submodule to same commit like in https://github.com/thesofproject/sof/commit/b37d466a69a8b8dfb0b1ef42ff164f87b005a4b8 (see comment in west.yml on this).

**Comment by marc-hb** [comment]
See better  https://github.com/thesofproject/sof/pull/7630

**Comment by jxstelter** [comment]
The same fixed in https://github.com/thesofproject/sof/pull/7630. Closing this one.

**Pull request closed without merging**